source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/mdp/rewards.py

```diff
@@ -104,3 +104,21 @@ def track_ang_vel_z_world_exp(
     asset = env.scene[asset_cfg.name]
     ang_vel_error = torch.square(env.command_manager.get_command(command_name)[:, 2] - asset.data.root_ang_vel_w[:, 2])
     return torch.exp(-ang_vel_error / std**2)
+
+
+def stand_still_error(env, command_name: str, asset_cfg: SceneEntityCfg = SceneEntityCfg("robot")) -> torch.Tensor:
+    """Penalize offsets from the default joint positions when the command is very small."""
+    asset = env.scene[asset_cfg.name]
+    command = env.command_manager.get_command(command_name)
+    # Penalize motion at zero commands
+    offset = asset.data.default_joint_pos[:, asset_cfg.joint_ids] - asset.data.joint_pos[:, asset_cfg.joint_ids]
+    return torch.sum(torch.abs(offset), dim=1) * (torch.norm(command[:, :2], dim=1) < 0.06)
```

Comment:
```suggestion
    command = env.command_manager.get_command(command_name)
    # Penalize motion at zero commands
    return mdp.joint_deviation_l1(asset_cfg) * (torch.norm(command[:, :2], dim=1) < 0.06)
```

Reply:
Makes sense to make threshold for command also a argument to the function as it depends on the "size" of the robot or how sensitive you want this term to be.